packages/hop-node/src/watchers/classes/L1Bridge.ts

```diff
@@ -295,4 +314,10 @@ export default class L1Bridge extends Bridge {
     )
     return address !== ethers.constants.AddressZero
   }
+
+  @rateLimitRetry
+  async getChallengePeriod (): Promise<number> {
+    const challengePeriod = await this.getReadBridgeContract().challengePeriod()
+    return challengePeriod
```

Comment:
Does this need to be casted to number type, ie `Number(challengePeriod.toString())`?

Reply:
Good catch. Updated with b5f414b3c40ab5bfeb96d099d57da95aef47e44a